static/js/popovers.js

```diff
@@ -81,8 +81,13 @@ function init_email_clipboard() {
 }
 
 function load_medium_avatar(user, elt) {
-    const user_avatar_url = "avatar/" + user.user_id + "/medium";
+    let user_avatar_url = "avatar/" + user.user_id + "/medium";
+    if (user.avatar_url && user.avatar_url.includes("version=")) {
+        const avatar_version = user.avatar_url.split("version=")[1];
+        user_avatar_url += "?v=" + avatar_version;
```

Comment:
Can you use the browser's `new URL` API to modify the URL in a valid way, rather than doing string-parsing?

Reply:
Is there a reason we don't just pass down `avatar_version` again, to avoid this parsing?

Reply:
No specific reason, I did like this because it was quicker. But if it's better the server gives to the client the avatar_version, I do it :)

Reply:
Just for curiosity, why are we avoiding parsing strings? Is it more error-prone?

Reply:
Yes, it's more error-prone, and some types of errors in doing string parsing of URLs can lead to security issues.